stdlib/pathname.rb

```diff
@@ -37,7 +44,7 @@ def cleanpath
   end
 
   def to_path
-    @path
+    @path.to_s
```

Comment:
what about doing this in the initialize, I think many methods below assume it's a string anyway